pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -2360,11 +2399,20 @@ void GSRendererHW::EmulateBlending(bool& DATE_PRIMID, bool& DATE_BARRIER, bool&
 	m_conf.ps.blend_b = ALPHA.B;
 	m_conf.ps.blend_c = ALPHA.C;
 	m_conf.ps.blend_d = ALPHA.D;
+	const bool force_a_one = !PRIM->ABE && AA1;
```

Comment:
I'd move it after the comment, also maybe replace the comment with AA1, and move the old one in the function.